Project-1/README.md

```diff
@@ -13,63 +13,300 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
 
 1. Provide the truth tables for each of the following propositional logic formulas. State whether each is a tautology, a contradiction, or contingent:
   ```
-  (a) (¬A→B)∨((A∧¬C)→B) 
-  (b) (A→B)∧(A→¬B)
-  (c) (A→(B∨C))∨(C→¬A) 
-  (d) ((A→B)∧C)∨(A∧D) 
+  (a) (¬A→B)∨((A∧¬C)→B) (Is a tautology)
+  
+  	A	B	C	((¬A → B) ∨ ((A ∧ ¬C) → B))
+		F	F	F	T
+		F	F	T	T
+		F	T	F	T
+		F	T	T	T
+		T	F	F	T
+		T	F	T	T
+		T	T	F	T
+		T	T	T	T
+	
+	
+  (b) (A→B)∧(A→¬B) (Is contingent)
+  	A	B	((A → B) ∧ (A → ¬B))
+		F	F	T
+		F	T	T
+		T	F	F
+		T	T	F
+	
+  (c) (A→(B∨C))∨(C→¬A) (Is a tautology)
+  
+  	A	B	C	((A → (B ∨ C)) ∨ (C → ¬A))
+		F	F	F	T
+		F	F	T	T
+		F	T	F	T
+		F	T	T	T
+		T	F	F	T
+		T	F	T	T
+		T	T	F	T
+		T	T	T	T
+		
+  (d) ((A→B)∧C)∨(A∧D) (Is contingent)
+  
+  	A	B	C	D	(((A → B) ∧ C) ∨ (A ∧ D))
+		F	F	F	F	F
+		F	F	F	T	F
+		F	F	T	F	T
+		F	F	T	T	T
+		F	T	F	F	F
+		F	T	F	T	F
+		F	T	T	F	T
+		F	T	T	T	T
+		T	F	F	F	F
+		T	F	F	T	T
+		T	F	T	F	F
+		T	F	T	T	T
+		T	T	F	F	F
+		T	T	F	T	T
+		T	T	T	F	T
+		T	T	T	T	T
+		
   ```
 	
 2. A _literal_ is an atomic formula or the negation of an atomic formula. We say a formula is in _conjunctive normal form_ (CNF) if it is the conjunction of the disjunction of literals. Find propositional logic formulas in CNF equivalent to each of the following:
   ```
   (a) (A→B)→C
+  	(A∨C)∧(¬B∨C)
+	
   (b) (A→(B∨C))∨(C→¬A)
+  	(A∨¬A) is  equivalent because tautologies are equivalent.
+  
   (c) (¬A∧¬B∧C)∨(¬A∧¬C)∨(B∧C)∨A 
+  	(A∨¬A) is  equivalent because tautologies are equivalent.
   ```
   
 3. Let V be the vocabulary of first-order logic consisting of a binary relation P and a unary relation F. Interpret P(x,y) as “x is a parent of y” and F(x) as “x is female.” Where possible define the following formulas in this vocabulary; where not possible, explain why: 
   
   ```
   (a)  B(x,y) that says that x is a brother of y  
+  	B(x,y)↔∃x∃y(~Fx & ∀z(P(z,x)⟷P(z,y) & ∃wP(w,x)) & x≠y) , but only if we accept that to be male is equivalent to the negation of being female, and if we 
```

Comment:
The variables in B(x,y) are implicitly universally quantified, but then you introduce existentially quantified x,y, so I'm unsure how to determine scope. This holds for each of the following formulas.